client/src/views/clan/ClanView.tsx

```diff
@@ -26,10 +26,11 @@ const ClanView = () => {
   const navigate = useNavigate();
   const Confirm = useConfirm();
   const { updateBrute } = useBrute();
-
+  // eslint-disable-next-line no-shadow
+  enum sortOptions { Default = 'default', Level = 'level', Rank = 'ranking', Victories = 'victories', Damage = 'damage'}
   const [clan, setClan] = useState<ClanGetResponse | null>(null);
   const [damageDisplayed, setDamageDisplayed] = useState(false);
-
+  const [sort, setSort] = React.useState<sortOptions>(sortOptions.Default);
```

Comment:
Ok, no, this whole system is not good, as first of all there are no translations, and this gets redefined on each render.

You need to declare the sort options before the component:
```ts
enum SortOption {
  default = 'default'
  ...
}
```

The use it in the useState like
```ts
const [sort, setSort] = useState<SortOption>(SortOption.default);
```

As a rule of thumb, if you feel forced to ignore an eslint rule, you're probably doing something wrong

Reply:
Oh you're right totally skipped it with my etwin issues

Only default was need it

For the enum, it's apparently a bug from Typescript , the solution is to add something in the eslint files, I don't know if I'm allow to do it.